de.fu_berlin.inf.dpp.ui.frontend/html/src/components/ToolBar/ConnectionSplitButton.jsx

```diff
@@ -14,37 +15,39 @@ class ConnectionSplitButton extends React.Component {
   constructor (props) {
     super(props)
     this.connectionStateMap = {
-      'INITIALIZING': { onClick: this.props.onConnect, messageId: 'action.connect' },
-      'NOT_CONNECTED': { onClick: this.props.onConnect, messageId: 'action.connect' },
-      'CONNECTED': { onClick: this.props.onDisconnect, messageId: 'action.disconnect' },
-      'ERROR': { onClick: noop, messageId: 'action.connectionError' },
-      'CONNECTING': { onClick: noop, messageId: 'action.connecting' },
-      'DISCONNECTING': { onClick: noop, messageId: 'action.disconnecting' }
+      'INITIALIZING': { onClick: this.props.onConnect, messageId: 'action.connect', icon: images.accountDisconnectedIcon },
```

Comment:
@AbdBarho I was not sure whether this creates a map containing the images as base64-string or simply as reference.